react/images/Check.js

```diff
@@ -1,26 +1,18 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Use from 'vtex.react-use-svg/Use'
 
 export default function Check({ size }) {
   return (
     <svg
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
This is always the same, why you don't put in Use?

Reply:
I think puting all of this in a single component would be a little strange. An idea that came in mind is to create another component called Svg, that wraps svg tags with default values for props xmlns and xmlnsXlink. Something like:

```
...
const Svg = ({ children, ...props}) => (
  <svg {...props}>
    { children }
  </svg>
);

Svg.defaultProps = {
  xmlns: 'http://www.w3.org/2000/svg',
  xmlnsXlink='http://www.w3.org/1999/xlink'
};
```

Using the component would be like: 
```
...
<Svg 
  width={size}
  height={size}
  viewBox="0 0 9 9"
  fill="none"
>
  <Use id="check" />
</Svg>
```

This Component would be useful in any other place that we use svgs. What do you think ?